frontend/Gruntfile.js

```diff
@@ -47,31 +42,37 @@ module.exports = function(grunt) {
 
     });
 
+
+    //run all tasks from directory /grunt-tasks
     grunt.loadTasks('grunt-tasks');
 
-    grunt.task.run('includeSource');
-    
-    grunt.registerTask('serve',[
+    //use this task for developing mode only
+    grunt.registerTask('serve', [
+        'clean:dev',
```

Comment:
I think that it is possible to do , but it can take a little more time for investigation.